pkg/netconfig/netutils.go

```diff
@@ -82,18 +82,26 @@ func (r *OSNetUtilsCommandExecutor) ExecuteIPTablesCommand(ipVersion string, arg
 	}
 	args = append([]string{"-w"}, args...)
 	cmd := exec.Command("ip"+ipVersion+"tables-"+r.ipTablesBackend, args...)
-	return cmd.Run()
+	return wrapCmd(cmd)
 }
 
 func (r *OSNetUtilsCommandExecutor) ExecuteIPSetCommand(args ...string) error {
 	cmd := exec.Command("ipset", args...)
-	return cmd.Run()
+	return wrapCmd(cmd)
```

Comment:
Just an idea: Could we maybe move the `execCommand` to `wrapCmd` and also return the output in the non error case? This way, we could use it in more places.

Reply:
we can do that but it'll get a little more complicated with the possibility to attach stdin to the cmd. I'll give it a go and let you decide.